addons/event/models/event_mail.py

```diff
@@ -133,20 +133,25 @@ def _warn_template_error(self, scheduler, exception):
             try:
                 event, template = scheduler.event_id, scheduler.template_id
                 emails = list(set([event.organizer_id.email, event.user_id.email, template.write_uid.email]))
-                subject = _("WARNING: Event Scheduler Error for event: %s" % event.name)
+                subject = _("WARNING: Event Scheduler Error for event: %s", event.name)
                 body = _("""Event Scheduler for:
-                              - Event: %s (%s)
-                              - Scheduled: %s
-                              - Template: %s (%s)
-
-                            Failed with error:
-                              - %s
-
-                            You receive this email because you are:
-                              - the organizer of the event,
-                              - or the responsible of the event,
-                              - or the last writer of the template."""
-                         % (event.name, event.id, scheduler.scheduled_date, template.name, template.id, ex_s))
+  - Event: %(event_name)s (%(event_id)s)
+  - Scheduled: %(date)s
+  - Template: %(template_name)s (%(template_id)s)
+
+Failed with error:
+  - %(error)s
+
+You receive this email because you are:
+  - the organizer of the event,
+  - or the responsible of the event,
+  - or the last writer of the template.""",
```

Comment:
Isn't the reindentation going to blow up the entire translation?

Reply:
Yes it does but on the other hand was not useful to indent in the first place. The translation has to be rewritten anyway as we are changing the placeholders.

Reply:
Good point and fair enough. Maybe add a final newline to the multiline string then?